modules/@apostrophecms/rich-text-widget/index.js

```diff
@@ -404,7 +505,9 @@ module.exports = {
               'href',
               'id',
               'name',
-              'target'
+              ...self.linkSchema
+                .filter(field => field.htmlAttribute)
+                .map(field => field.name)
```

Comment:
`field.htmlAttribute` should be the actual attribute name (if any). There is no requirement that it match the schema property name, and if sometimes won't because camelCase is recommended for field names and HTML generally is kebab-base.